Classes/Emogrifier.php

```diff
@@ -1756,6 +1766,20 @@ public function handleXpathError($type, $message, $file, $line, array $context)
             );
         }
 
+        // Catches eventual warnings generated by invalid selectors in method getNodesToExclude()
+        if ($type === E_WARNING && isset($context['selectorToExclude'])) {
```

Comment:
This can be unified to a `if ($type === E_WARNING)` around both cases.

Reply:
Actually, since the `handleXpathError()` method is only assigned to the E_WARNING type, I chose to get rid of the `if ($type === E_WARNING)` conditions.

Maybe the `handleXpathError()` method should be renamed to better reflect its scope on warnings only? What do you think?

Reply:
Had to add back condition `if ($type === E_WARNING)` (although it's always true), because phpcs was raising a warning because of parameter `$type` not being used anymore. Is there a better solution here?

Reply:
Yes, please let's rename it.

Reply:
As for the unused parameter, you can add an `@codingStandardsIgnoreLine` anotation to the line above it.